homeassistant/components/toon/__init__.py

```diff
@@ -1,142 +1,195 @@
 """Support for Toon van Eneco devices."""
-from datetime import datetime, timedelta
 import logging
+from typing import Any, Dict
 
 import voluptuous as vol
 
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
-import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.discovery import load_platform
-from homeassistant.util import Throttle
+from homeassistant.helpers import (config_validation as cv,
+                                   device_registry as dr)
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
 
-REQUIREMENTS = ['toonlib==1.1.3']
+from . import config_flow  # noqa  pylint_disable=unused-import
+from .const import (
+    CONF_CLIENT_ID, CONF_CLIENT_SECRET, CONF_DISPLAY, CONF_TENANT,
+    DATA_TOON_CLIENT, DATA_TOON_CONFIG, DOMAIN)
 
-_LOGGER = logging.getLogger(__name__)
-
-CONF_GAS = 'gas'
-CONF_SOLAR = 'solar'
-
-DEFAULT_GAS = True
-DEFAULT_SOLAR = False
-DOMAIN = 'toon'
-
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
+REQUIREMENTS = ['toonapilib==3.0.7']
```

Comment:
Please use 3.0.9 as 3.0.7 is broken for 3.5 compatibility

Reply:
Thanks! 👍 
Tested locally and working like a charm. Upped version of the `toonapilib` dep.